test-butler-demo/src/androidTest/java/com/linkedin/android/testbutler/demo/ImmersiveModeConfirmationDisablerTest.java

```diff
@@ -0,0 +1,26 @@
+package com.linkedin.android.testbutler.demo;
+
+import android.support.test.rule.ActivityTestRule;
+import com.linkedin.android.testbutler.TestButler;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+
+public class ImmersiveModeConfirmationDisablerTest {
+    @Rule public ActivityTestRule<MainActivity> testRule = new ActivityTestRule<>(MainActivity.class);
+
+    @Test
+    public void immersiveModeConfirmationDoesNotShow() {
```

Comment:
I'm guessing there's not a good way to test that the confirmation DOES appear if you set this to `true`?